src/main/java/com/avparamonov/checkers/services/BoardService.java

```diff
@@ -20,36 +17,139 @@
 @Service
 public class BoardService {
 
-    @Autowired
-    private BoardRepository boardRepository;
-
-    @Autowired
-    private CellService cellService;
-
-    @Autowired
-    private CheckerService checkerService;
-
-    public Board initBoard() {
-        List<Cell> cells = IntStream.range(1, 9)
-                .mapToObj(r -> IntStream.range(1, 9)
-                        .mapToObj(c -> {
-                            Cell cell = cellService.createInPosition(r, c);
-                            if(((r == 1 | r == 3) && c % 2 != 0) || (r == 2 && c % 2 == 0)) {
-                                cell.setChecker(checkerService.create(Side.BLACK));
-                                cell.setOccupied(true);
-                            } else if(((r == 6 | r == 8) && c % 2 == 0) || (r == 7 && c % 2 != 0)) {
-                                cell.setChecker(checkerService.create(Side.RED));
-                                cell.setOccupied(true);
+    private enum Directions {
```

Comment:
Please move this enum to a separate class, it's hard to understand why it's included to some service